experimental/packages/exporter-trace-otlp-grpc/src/OTLPTraceExporter.ts

```diff
@@ -59,7 +57,7 @@ export class OTLPTraceExporter
       ? validateAndNormalizeUrl(getEnv().OTEL_EXPORTER_OTLP_TRACES_ENDPOINT)
       : getEnv().OTEL_EXPORTER_OTLP_ENDPOINT.length > 0
       ? validateAndNormalizeUrl(getEnv().OTEL_EXPORTER_OTLP_ENDPOINT)
-      : DEFAULT_COLLECTOR_URL;
+      : validateAndNormalizeUrl(DEFAULT_COLLECTOR_URL);
```

Comment:
Does the default URL need to be validated and normalized?

Reply:
Yes because the `DEFAULT_COLLECTOR_URL` didn't have a scheme so we didn't need to pass it to `validateAndNormalizeUrl` like we do when user adds a url to `config.url` or when using env vars. 

Now that I updated `DEFAULT_COLLECTOR_URL` to have an `http` scheme (like we discussed in the sig so it matches the spec) we need to pass it to `validateAndNormalizeUrl` to remove the scheme. 

That's my understanding of it but please correct me if I'm wrong here. :)

Reply:
@dyladan Do you have any questions on the above explanation?